Missing @radix-ui/react-slider Dependency

**Bug: Missing @radix-ui/react-slider Dependency**

## Summary

The slider.tsx component fails to compile due to a missing dependency. The component imports `@radix-ui/react-slider` but this package is not installed in the project.

## Error Details

```
Cannot find module '@radix-ui/react-slider' or its corresponding type declarations.
```

<img width="1422" height="187" alt="Image" src="https://github.com/user-attachments/assets/f606e476-cf6c-437f-9384-b95a3693329e" />

## Affected File

```
src/components/ui/8bit/slider.tsx
```

## Code Reference

```typescript
import * as SliderPrimitive from "@radix-ui/react-slider";
```

Line 8 in slider.tsx attempts to import the Radix UI Slider primitive.

## Root Cause

The Radix UI Slider primitive package is referenced in the component but not listed in the project's package.json dependencies.

## Impact

- ❌ TypeScript compilation errors
- ❌ Component cannot be used in the application
- ❌ Build process will fail
- ❌ Development server may crash

## Solution

Install the missing dependency manually:

**npm:**
```bash
npm install @radix-ui/react-slider
```

**yarn:**
```bash
yarn add @radix-ui/react-slider
```

**pnpm:**
```bash
pnpm add @radix-ui/react-slider
```

--- a/registry.json
+++ b/registry.json
@@ -829,8 +829,7 @@
       "type": "registry:component",
       "title": "8-bit Slider",
       "description": "A simple 8-bit slider component",
-      "registryDependencies": [],
-      "dependencies": ["@radix-ui/react-slider"],
+      "registryDependencies": ["slider"],
       "files": [
         {
           "path": "components/ui/8bit/slider.tsx",

